web/card-fields/multiple-card-fields.html

```diff
@@ -1,115 +1,356 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>PayPal Multi Card Fields | Client Example</title>
 
-        const onApprove = (data) => {
-            console.warn('Payment Approved:', data)
-        };
+    <style>
+      * {
+        box-sizing: border-box;
+        font-family: Helvetica;
+      }
 
-        let cardNumberFieldValid = false;
+      .container {
+        width: 445px;
+        margin-bottom: 10px;
+      }
 
-        const setValidNumberStyle = () => {
-            cardNumberContainer.style.border = '1px solid #333';
-        };
+      .card-field-container {
+        border: 1px solid #909697;
+        border-radius: 3px;
+        line-height: 0px;
+        min-height: 35px;
+        display: none;
+        height: 62px;
+      }
 
-        const setInvalidNumberStyle = () => {
-            cardNumberContainer.style.border = '1px solid red';
-        };
+      .valid {
+        border: 1px solid #909697;
+      }
+      .invalid {
+        border: 1px solid red;
+      }
 
-        let cardCVVFieldValid = false;
+      button {
+        width: 100%;
+        height: 48px;
+        background: #000000;
+        border-radius: 25px;
+        color: white;
+        font-size: 18px;
+        border-color: #000000;
+        cursor: pointer;
+      }
+      .errors-container {
+        display: none;
+        margin-bottom: 10px;
+      }
+      .error-icon {
+        color: red;
+      }
+      .loading {
+        transform: translateX(0);
+        animation: 1.5s loading-placeholder ease-in-out infinite;
+        cursor: not-allowed;
+        opacity: 0.5;
+      }
+      @keyframes loading-placeholder {
+        0% {
+          opacity: 0.1;
+        }
+        50% {
+          opacity: 1;
+        }
+        100% {
+          opacity: 0.1;
+        }
+      }
+    </style>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
+  </head>
 
-        const setValidCVVStyle = () => {
-            cardCVVContainer.style.border = '1px solid #333';
-        };
+  <body>
+    <!-- PayPal JS SDK -->
+    <script src="https://www.paypal.com/sdk/js?client-id=test&components=card-fields,buttons&debug=true&intent=capture"></script>
 
-        const setInvalidCVVStyle = () => {
-            cardCVVContainer.style.border = '1px solid red';
-        };
+    <h2>PayPal Multiple Card Fields</h2>
+    <div id="multi-transactional-card-field">
+        <div id="card-number-field-container" class="container card-field-container"></div>
+        <div id="errors-number-container" class="errors-container">
+          <i class="fas fa-exclamation-triangle error-icon"></i>
+          <span id="errors-number"></span>
```

Comment:
I think we should use `<div>` instead of `<span>` since it's `block` by default.
```suggestion
          <div id="errors-number"></div>
```